PRChecker/macOS/Unique Views/MenuView.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  MenuView.swift
+//  PRChecker (macOS)
+//
+//  Created by Chen Yuhan on 2021/11/07.
+//
+
+import SwiftUI
+
+struct MenuView: View {
+
+    @Environment(\.openURL) var openURL
+
+    @StateObject var prListViewModel = PRListViewModel()
+    
+    private let maxPRCount = 5
+
+    var body: some View {
+        List(prListViewModel.prList.prefix(maxPRCount)) { pullRequest in
+            MenuBarPRCell(pullRequest: pullRequest)
+                .onTapGesture {
+                    openURL(URL(string: pullRequest.url)!)
+                }
+        }
+        .onAppear {
+            prListViewModel.getPRList()
```

Comment:
Does it refresh as-is? Or will that be a later PR?

Reply:
No, should we implement pull-to-refresh here?

Reply:
I was thinking maybe a timer for the menu item... Pull to Refresh would work too though

Reply:
Indeed, a timer would be better, then we can show&update the count of PR in menu bar.
Let me add this spec to the TODO list.